py3status/modules/uptime.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed, formatted with predefined
+          placeholders; defaults to "{days} days, {hours} hours, {minutes} min"
+
+Available placeholders in format string:
+        - {days}: days part of the uptime
+        - {hours}: hours part of the uptime
+        - {minutes}: minutes part of the uptime
+        - {seconds}: seconds part of the uptime
+        - No {year} provided !
```

Comment:
I'm actually have `years, up = divmod(up, 31536000) # 365 days year` in my code. For the hibernate part, that's up to the system to handle this as it wants, I'm just reading `/proc/uptime` :)

Anyway, forget about the 365.25 days. I'm getting rid of the floating point from `/proc/uptime` since divmod doesn't seem to call `math.floor` and I don't want to make an `int()` cast after each call (and can't do it when calling `format` as I did until now since, with conditionnal computing of differents granularity, it would be trying to do things like `int(None)` without conditionnal formatting)